### DISCUSSION TITLE
No indication of which actions are disabled in the action list

### DISCUSSION THREAD

**Discussion by kisielk** [community/community#56910] [Actions]
### Select Topic Area

Product Feedback

### Body

When you visit https://github.com/$ORG/$REPO/actions there is no indication in the list of workflows if any of the workflows have been disabled. It should be indicated somehow in the UI so it's clear which, if any, of the workflows listed have been disabled without having to click on each one to check.

**Comment by mathbunnyru** [comment]
I strongly agree with this feature.
I think the reasonable approaches:
- grey out the disabled workflows
- add an icon indicating the workflow is disabled
- move disabled workflows to separate list on the left side of the page

I think the last approach is the best - people might change names of workflows, because the features of the workflows change and it would be nice to not have a list of all workflows which existed in the repo.
An example where it doesn't look nice: https://github.com/jupyter/docker-stacks/actions

Another improvement would be to be to move callable (which can't be run directly) to exclude from the workflows section, because they can't be called and there are no "runs" for this type of workflows.

> No indication of which actions are disabled in the action list

I think it should be workflows, not actions
